doc/tutorials/application-integration.md

```diff
@@ -12,10 +12,15 @@ In Elektra different forms of application integrations are possible:
    - [Intercept File System](/src/bindings/intercept/fs/README.md)
 3. Integration where applications directly use Elektra to read and
    store settings.
+   1. Using the low-level API.
```

Comment:
Is it possible to call it "i" and "ii" here?

Reply:
No, markdown only recognises Arabic numbers as list items. GitHub is quite unique in how it renders nested numbered lists. Most other markdown renderers I tried render all levels with Arabic numerals.

I don't know what we use for the website, but I'll change the references in the text below to `3.1` and `3.2`. IMO it is easier to understand that `3.1` references `3.i` than the other way around.

Reply:
Yes, I agree. Only Arabic numbers is easier.